maxi-blocks/maxi-blocks
# Issue 3890: Fixed multiple Appender issue

# Description
Fixed multiple appenders overlapping issues by implementing drop down for multiple appenders 


Fixes #3890

# How Has This Been Tested?
1. Add accordion inside column maxi and check the appender dropdown list to add blocks to accordion maxi and also column maxi.
2. Check the appender overlapping issue by adding container blocks (column, accordion, group) into other container blocks. 


# Test checklist



**_ Front/Back Testing _**


**_ Pre-Code Testing _**


# Checklist
